pandas/core/indexes/category.py

```diff
@@ -102,13 +101,13 @@ class CategoricalIndex(Index, accessor.PandasDelegate):
     See Also
     --------
     Index : The base pandas Index type.
-    Categorical : A categorical variable in classic R / S-plus fashion.
+    Categorical : A categorical array.
     CategoricalDtype : Type for categorical data.
 
     Notes
     -----
     See the `user guide
-    <https://pandas-docs.github.io/pandas-docs-travis/advanced.html#categoricalindex>`_
+    <http://pandas.pydata.org/pandas-docs/stable/user_guide/advanced.html#categoricalindex>`_
```

Comment:
pandas.pydata.org supports https, shouldn't that be used instead?

OTOH, I can see that in the code base we use a mix of **http**://pandas.pydata.org and **https**://pandas.pydata.org, so such a change could also be a seperate commit, where this is doe for the whole code base.

Reply:
Yea separate PR is fine